docs/kh2/file/type/mset.md

```diff
@@ -0,0 +1,81 @@
+# [Kingdom Hearts II](../../index.md) - MSET (Moveset)
+
+The MSET files contain animations and the effects related to them (I-frames, hitboxes, effects...)
+It is a [BAR file](bar.md) that contains a list of "slots". Each slot may or may not have a "motion".
+The various actions in the game point to a specific motion by a slot position in this list, thus many of the motions are dummies, since the characters only use some of them.
+Eg: For Sora, slot 0 is idle, slot 4 is walking and slot 604 is the basic attack.
+
+Each motion is a BAR file with 2 entries, the model animation and the effects.
```

Comment:
When you unpack a motion entry in a MSET BAR I can find two files: a small one and a big one. The big one I imagine is the actual motion file that moves a 3D model. The small one (I am looking at a file of 62 bytes long now) is the effect structure you are mentioning here?

Reply:
Yes, that's right. They are always in that order. I changed "animation and effects" to "animation followed by effects" to make it clearer.